### PR TITLE
Refactor: Fix warning SA1101

### PR DESCRIPTION
**Pull request by AdamPlaskitt**
Related to #202

Prefix local calls with this.
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1101.md

### CODE REVIEW

**Comment by JamieMagee** [comment]
Thanks for the PR 😄

**Comment by amitla1** [comment]
@AdamPlaskitt  the warnings in the build need to be fixed

**Comment by AdamPlaskitt** [comment]
> @AdamPlaskitt the warnings in the build need to be fixed and then I can approve

Hey @amitla1,
I can see any build warnings in the files I changed?

**Comment by amitla1** [comment]
> > @AdamPlaskitt the warnings in the build need to be fixed and then I can approve
> 
> Hey @amitla1, I can see any build warnings in the files I changed?

Yes it's not a file you changed but it's related to the fix SA1101 that this branch was for: 
image

File: src/Microsoft.ComponentDetection.Detectors/nuget/NuGetComponentDetector.cs [hunk 1]
```diff
@@ -140,15 +140,15 @@ private void ParsePaketLock(ProcessRequest processRequest)
...
+            using var reader = new StreamReader(stream.Stream);
 
             string line;
             while ((line = reader.ReadLine()) != null)
             {
                 var matches = Regex.Matches(line, @"\s*([a-zA-Z0-9-.]*) \([<>=]*[ ]*([0-9a-zA-Z-.]*)\)", RegexOptions.Singleline);
                 foreach (Match match in matches)
                 {
-                    string name = match.Groups[1].Value;
+                    var name = match.Groups[1].Value;
```
Comment (amitla1) [inline src/Microsoft.ComponentDetection.Detectors/nuget/NuGetComponentDetector.cs:151]: What was the reason behind changing string to var here?

**Review by JamieMagee — APPROVED** [review]

**Pull request merged**